plugins/roadie-backstage-entity-validator/src/relativeSpaceValidation.ts

```diff
@@ -48,12 +48,12 @@ const validateTechDocs = async (data, filePath) => {
 };
 
 export const relativeSpaceValidation = async (
-  fileContents,
-  filePath,
-  verbose,
+  fileContents: string,
+  filePath: string,
+  verbose: boolean,
 ) => {
   try {
-    const data = yaml.loadAll(fileContents, { schema: yaml.CORE_SCHEMA });
+    const data = yaml.loadAll(fileContents, null, { schema: yaml.CORE_SCHEMA });
```

Comment:
This seems like a material change? Or maybe a new version of the yaml lib requires this?

Reply:
I think loadAll expected the options in the 3rd positional paramter but the file was regular js so TS didnt check and now that I converted it to TS it complains that it should be 3rd 🤔 So I think it's a fix actually

Reply:
ok.. 👍  Hopefully the tests will catch it if its broken.